apps/smart-contracts/core/test/DepositHook.test.ts

```diff
@@ -349,39 +349,12 @@ describe('=> DepositHook', () => {
       )
     })
 
-    it('should be settable to an address', async () => {
-      expect(await depositHook.getCollateral()).to.eq(ZERO_ADDRESS)
-
-      await depositHook.connect(deployer).setCollateral(collateral.address)
-
-      expect(await depositHook.getCollateral()).to.eq(collateral.address)
-    })
-
-    it('should be settable to the zero address', async () => {
-      await depositHook.connect(deployer).setCollateral(collateral.address)
-      expect(await depositHook.getCollateral()).to.eq(collateral.address)
-
-      await depositHook.connect(deployer).setCollateral(ZERO_ADDRESS)
-
-      expect(await depositHook.getCollateral()).to.eq(ZERO_ADDRESS)
-    })
-
-    it('should be settable to the same value twice', async () => {
-      expect(await depositHook.getCollateral()).to.eq(ZERO_ADDRESS)
-
-      await depositHook.connect(deployer).setCollateral(collateral.address)
-
-      expect(await depositHook.getCollateral()).to.eq(collateral.address)
-
-      await depositHook.connect(deployer).setCollateral(collateral.address)
-
-      expect(await depositHook.getCollateral()).to.eq(collateral.address)
-    })
-
-    it('emits CollateralChange', async () => {
-      const tx = await depositHook.connect(deployer).setCollateral(collateral.address)
+    it('succeeds if role holder', async () => {
+      expect(
+        await depositHook.hasRole(await depositHook.SET_COLLATERAL_ROLE(), deployer.address)
+      ).to.eq(true)
 
-      await expect(tx).to.emit(depositHook, 'CollateralChange').withArgs(collateral.address)
+      await expect(depositHook.connect(deployer).setCollateral(collateral.address))
```

Comment:
```suggestion
      await expect(depositHook.connect(deployer).setCollateral(collateral.address))
```
Are expects even necessary in these tests? I think we can just call them no? And if they revert then the testcase fails.